docs/network-spec/connection-manager.tex

```diff
@@ -92,15 +92,19 @@ \chapter{Connection Manager State Machine Specification}
 \def\TerminatingState{\texttt{TerminatingState\textsuperscript{$\tau$}}}
 \def\TerminatedState{\texttt{TerminatedState}}
 
-% Connection manager's ransitions
+% Connection manager's transitions
 \def\Reserve{\textsf{Reserve}}
 \def\Connected{\textsf{Connected}}
 \def\Accepted{\textsf{Accepted}}
 \def\Overwritten{\textsf{Overwritten}}
+\def\SelfConnA{\textsf{SelfConn}}
+\def\SelfConnB{\textsf{SelfConn}\textsuperscript{\textsf{-1}}}
```

Comment:
I'd suggest `\SelfConn` and `\SelfConfInv`.

Reply:
We should also explain these transitions in a subsection as we do with the other ones. I'd mentioned that connection manager is not trying to reliably detect such connections since that's impossible as inbound and outbound addresses can be different which depends on external factors, but it just reacts to events.  Self connections will likely be dropped in a longer time frame by the churn policy as they only add latency to delivered headers or blocks.

Reply:
> I'd suggest `\SelfConn` and `\SelfConfInv`.

There's already connections with those names so I chose to disambiguate.

Reply:
hmm it seems you documented these transitions but didn't add them to the diagram so I will rename them to the right names

Reply:
> There's already connections with those names so I chose to disambiguate.

So now we have `\SelfConn`, `\SelfConnInf`, `\SelfConnA` and `\SelfConnB`?

Reply:
We have `\SelfConn` and `\SelfConnRev` - if they are the same, there's no reason to disambiguate; it can be confusing.

Reply:
We have `\SelfConn` and `\SelfConnRev` and `\SelfConnPrim`